### PR TITLE
fix: getDefaultLineHeight should return default font family line height for unknown font

### PR DESCRIPTION
**Pull request by CoyoteWaltz**
## Title

fix: `getDefaultLineHeight` always returns a default

## Why

The reason why I want to fix the `getDefaultLineHeight`'s return value is that after syncing with the latest `origin/master`, my forked excalidraw with my custom fonts app lost all text elements XD.

After doing some research, I found the `lineHeight` of the custom fonts is missing. Therefore, I prefer that `getDefaultLineHeight` always returns a value.

BTW: I forked the **AMAZING** excalidraw and added some custom fonts which is implemented at my [branch](https://github.com/CoyoteWaltz/excalidraw/tree/feat-addFonts/src/fork_by_coyote).

ALSO I wonder if there exists an elegant way to extend custom fonts with **minimal changes** and **better adoption with upcoming upper stream features**.

Thanks!

### CODE REVIEW

File: src/element/textElement.ts [hunk 1]
```diff
@@ -888,7 +888,10 @@ const DEFAULT_LINE_HEIGHT = {
 
 export const getDefaultLineHeight = (fontFamily: FontFamilyValues) => {
   if (fontFamily) {
-    return DEFAULT_LINE_HEIGHT[fontFamily];
+    return (
```
Comment (ad1992) [inline src/element/textElement.ts:891]: @CoyoteWaltz could you also add a test for this?

Comment (CoyoteWaltz) [reply src/element/textElement.ts:891]: test added, pls check

File: src/element/textElement.test.ts [hunk 2]
```diff
@@ -332,6 +332,10 @@ describe("Test getDefaultLineHeight", () => {
     //@ts-ignore
     expect(getDefaultLineHeight()).toBe(1.25);
   });
+  it("should return line height using default font family when passing a unknown font", () => {
```
Comment (ad1992) [inline src/element/textElement.test.ts:335]:
```suggestion

  it("should return line height using default font family when passing an unknown font", () => {
```

File: src/element/textElement.test.ts [hunk 3]
```diff
@@ -332,6 +332,11 @@ describe("Test getDefaultLineHeight", () => {
     //@ts-ignore
     expect(getDefaultLineHeight()).toBe(1.25);
   });
+
+  it("should return line height using default font family when passing an unknown font", () => {
+    const UNKNOWN_FONT = -1;
+    expect(getDefaultLineHeight(UNKNOWN_FONT)).toBe(1.25);
+  });
   it("should return correct line height", () => {
```
Comment (ad1992) [inline src/element/textElement.test.ts:340]:
```suggestion

  it("should return correct line height", () => {
```

File: src/element/textElement.test.ts [hunk 4]
```diff
@@ -332,6 +332,12 @@ describe("Test getDefaultLineHeight", () => {
     //@ts-ignore
     expect(getDefaultLineHeight()).toBe(1.25);
   });
+
+  it("should return line height using default font family when passing an unknown font", () => {
```
Comment (ad1992) [inline src/element/textElement.test.ts:336]:
```suggestion
  it("should return line height using default font family for unknown font", () => {
```

File: src/element/textElement.test.ts [hunk 5]
```diff
@@ -332,6 +332,12 @@ describe("Test getDefaultLineHeight", () => {
...
+    const UNKNOWN_FONT = -1;
```
Comment (ad1992) [inline src/element/textElement.test.ts:337]:
```suggestion
    const UNKNOWN_FONT = 5;
```
This is just for simplicity so its a positive key and not to be confused with negative index when checking test.

**Review by ad1992 — APPROVED** [review]
LGTM, thanks @CoyoteWaltz

**Pull request merged**